habitat_baselines/rl/ddppo/policy/resnet_policy.py

```diff
@@ -77,25 +78,32 @@ def __init__(
                 normalize_visual_inputs=normalize_visual_inputs,
                 force_blind_policy=force_blind_policy,
                 discrete_actions=discrete_actions,
+                fuse_keys=fuse_keys,
+                include_visual_keys=include_visual_keys,
             ),
-            dim_actions=dim_actions,  # for action distribution
+            dim_actions=get_num_actions(action_space),
             policy_config=policy_config,
         )
 
     @classmethod
     def from_config(
-        cls, config: Config, observation_space: spaces.Dict, action_space
+        cls,
+        config: Config,
+        observation_space: spaces.Dict,
+        action_space,
     ):
+        any_rgb = any(["rgb" in k for k in observation_space.spaces])
```

Comment:
```suggestion
        any_rgb = any("rgb" in k for k in observation_space.spaces)
```
Any takes a generator.